packages/firebase_auth/firebase_auth/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.15.5+2
+
+* Update the platform interface dependency to 1.1.7.
```

Comment:
Can you include the test change in the CHANGELOG?

Reply:
Done.